### ISSUE TITLE
emotion prob extract

### ISSUE THREAD

**Issue by JingRH** [FunAudioLLM/SenseVoice#228]
If I want to obtain the probabilities of 8 emotions:

arduino
複製
編輯
"<|HAPPY|>",
"<|SAD|>",
"<|ANGRY|>",
"<|NEUTRAL|>",
"<|FEARFUL|>",
"<|DISGUSTED|>",
"<|SURPRISED|>",
"<|OTHER|>",
should I take the logits at the corresponding token ID positions from the second frame?

`        for i in range(b):
            x = ctc_logits[i, : encoder_out_lens[i].item(), :]
            yseq = x.argmax(dim=-1)

            #todo
            emotion_logits=x[1,:]
`
Can I do this?

**Comment by kmvolv** [comment]
Wanted to obtain the same, commenting to know whether you were able to figure it out?
